dist/tools/radvd/radvd.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/sh
+
+CURRENT_DIR="$(dirname $(readlink -f $0))"
+PIDFILE="/tmp/radvd-${SUDO_USER}.pid"
+
+stop_radvd() {
+    if [ -f "${PIDFILE}" ]; then
+        PID=$(cat ${PIDFILE})
+    fi
+    if [ -n "${PID}" ]; then
+        kill ${PID}
+        rm ${PIDFILE}
+    fi
+}
+
+start_radvd() {
+    export TAP
+    export PREFIX
+    cat ${CURRENT_DIR}/radvd.conf | envsubst | radvd -C /dev/stdin -u ${SUDO_USER} -p ${PIDFILE}
```

Comment:
Not sure this is just a style thing, but
```suggestion
    radvd -C <(cat ${CURRENT_DIR}/radvd.conf | envsubst) -u ${SUDO_USER} -p ${PIDFILE}
```

seems cleaner to me.

Reply:
`/bin/sh` doesn't like `<(…)`:

    dist/tools/radvd/radvd.sh: 19: Syntax error: "(" unexpected

Reply:
Ok.